src/main/java/net/syscon/elite/service/CourtHearingsService.java

```diff
@@ -56,6 +57,7 @@ public CourtHearingsService(final OffenderBookingRepository offenderBookingRepos
     }
 
     @Transactional
+    @VerifyBookingAccess
     public CourtHearing scheduleHearing(final Long bookingId, final PrisonToCourtHearing hearing) {
```

Comment:
Probably will want the `@HasWriteScope`

Reply:
On the resource you'll also want `@ProxyUser`

Reply:
Done